### ISSUE TITLE
Map converting doesn't work

### ISSUE THREAD

**Issue by NickMarkick** [wootguy/SevenKewp#29]
![image](https://github.com/user-attachments/assets/3119d188-c51a-428f-9a90-ab56d50b3577)

Did all the steps correctly I think (Compiled the programs that were needed to be compiled), by the way the tutorial mentioned nothing about needing to have the resguy.exe and the resguy_default_content.txt files

**Comment by NickMarkick** [comment]
![image](https://github.com/user-attachments/assets/7a89ff9c-3d39-4833-b0e6-cd84827c4837)

**Comment by wootguy** [comment]
I got different output downloading [from here](http://scmapdb.wikidot.com/map:pizza-ya-san):

```
Lowercasing files...DONE
Merging external models
Converting model event sounds
ERROR: Model models\pizza_ya_san\p_9mmhandgun.mdl has invalid texture (glock1.BMP 572x288)
ERROR: Model models\pizza_ya_san\p_glock18jet.mdl has invalid texture (glock1.bmp 572x288)
ERROR: Model models\pizza_ya_san\v_9mmhandgun.mdl has invalid texture (glock1.BMP 588x296)
ERROR: Model models\pizza_ya_san\v_glock18jet.mdl has invalid texture (glock1.bmp 572x288)
ERROR: Model models\pizza_ya_san\w_9mmhandgun.mdl has invalid texture (glock1.BMP 572x288)
ERROR: Model models\pizza_ya_san\w_glock18jet.mdl has invalid texture (glock1.bmp 572x288)

Searching for incompatible entity settings...
Progress: 1 / 2  (pizza_ya_san1.bsp)
Invalid WAV format: takedeppo/50cal_fire.wav (2 chan 48000 16-bit)
Suboptimal WAV format: takedeppo/50cal_fun_time.wav (1 chan 8000 16-bit)
ffmpeg -i sound/takedeppo/50cal_fire.wav -c:a pcm_u8 -ac 1 -ar 22050 -y sound/takedeppo/50cal_fire_v2.wav
ffmpeg -i sound/takedeppo/50cal_fun_time.wav -c:a pcm_u8 -ac 1 -ar 8000 -y sound/takedeppo/50cal_fun_time_v2.wav
```
neither ripent nor bspguy touched the bsp, so idk how yours got corrupted. I'll update the instructions for resguy.

**Comment by NickMarkick** [comment]
I forgot to mention that I changed the models to fix them, I downloaded the map from the same source

**Comment by NickMarkick** [comment]
Also I think it didn't even touch the map because modified date didn't change

**Comment by NickMarkick** [comment]
I also checked the supported sven coop maps and it says that this map is supported so I have no idea why it doesn't convert properly

**Comment by wootguy** [comment]
yeah, you should be able to load `pizza_ya_san1.bsp` from the scmapdb download, without running the conversion script.

You might have multiple versions of the bsp installed (for example, one in `valve/maps/` and one in `valve_downloads/maps`). If not, attach the bsp being loaded here.

**Comment by NickMarkick** [comment]
Yeah I have the bsp in both directories, but its the same one
[pizza_ya_san1.zip](https://github.com/user-attachments/files/17594309/pizza_ya_san1.zip)

**Comment by wootguy** [comment]
Alright, I'm stumped now. That loaded for me. If you're willing, send your entire dedicated server install and I'll take a look.

**Comment by NickMarkick** [comment]
Alright

**Comment by NickMarkick** [comment]
Here: https://drive.google.com/file/d/1K1QUTU_7gLS-BQl6fdVEGoYYhgWCLfhP/view?usp=sharing

**Comment by wootguy** [comment]
What command do you use to launch the server? This is working for me, and I can see AMXX is loaded:
`hlds.exe +map pizza_ya_san1 -num_edicts 2047`

I got a crash without `-num_edicts`, but nothing about an invalid bsp version.

**Comment by NickMarkick** [comment]
Here's mine: "D:\Games\Steam\steamapps\common\hlds\hlds.exe -dll addons/metamod/metamod.dll -console +map pizza_ya_san1 +mp_allowmonsters 1 +maxplayers 4 +sv_cheats 0 -pingboost 2 -debug -autoupdate -insecure -num_edicts 8192"

**Comment by NickMarkick** [comment]
Tried yours but I still get the same error

**Comment by wootguy** [comment]
wtf that also works. Idk what else could be different between our setups. Windows version maybe? I'm on 10.

![image](https://github.com/user-attachments/assets/716436b2-1c20-439e-a6d7-b61e8b483066)

**Comment by NickMarkick** [comment]
I'm on windows 11 24h2

**Comment by NickMarkick** [comment]
> wtf that also works. Idk what else could be different between our setups. Windows version maybe? I'm on 10.
> 
> ![image](https://private-user-images.githubusercontent.com/12087544/382153764-716436b2-1c20-439e-a6d7-b61e8b483066.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0MjQyOTAsIm5iZiI6MTczMDQyMzk5MCwicGF0aCI6Ii8xMjA4NzU0NC8zODIxNTM3NjQtNzE2NDM2YjItMWMyMC00MzllLWE2ZDctYjYxZThiNDgzMDY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAxVDAxMTk1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2YjQ3YTdiYTQ0Mjc1ZTQ0OTEzYTA4OWM3MTUzMjg2ZmM4YmY5YjdiYTZjMjJiZmFmMTU2N2UwYzM0YjdhZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jOrQBIP6htuojT4A1rBJwX1Ndyx831R5nn8Rx81TSAo)

Does it actually work in game though

**Comment by wootguy** [comment]
I can join but the map plugin isn't loaded, so the custom weapons don't work (a separate issue). I'll make a Win11 VM set up to test this.

**Comment by NickMarkick** [comment]
Yeah I wanted to ask about the map plugin thing, I compiled the plugin and I put it in the maps folder but from what you're saying it doesn't seem to work

**Comment by wootguy** [comment]
The plugin goes here: `valve/plugins/maps/pizza_ya_san.dll` and then you need to add `map_plugin pizza_ya_san` to `pizza_ya_san1.cfg`

**Comment by NickMarkick** [comment]
Oh alright thanks

**Comment by wootguy** [comment]
`valve` and `valve_downloads` are interchangeable, but the map/plugin files should only exist in one of those folders. I forget which one has priority. I think `valve` is searched first.

**Comment by NickMarkick** [comment]
Alright

**Comment by wootguy** [comment]
Bad news, it still works with a fresh install of Windows 11. I had to install Visual Studio before the mod would load, but I don't get a BSP version error.

**Comment by NickMarkick** [comment]
What version of Half-Life 1 are you using?

**Comment by wootguy** [comment]
I use the latest HL client from steam, no betas.

...and now I've just realized you posted a client error. I thought the server was complaining about a BSP version, but the console doesn't have a green background.

Try copying the server's bsp file to your client and try again. Maybe something went wrong when you downloaded from the server.

**Comment by NickMarkick** [comment]
Oh I'm using the steam legacy client, maybe it'll work on the 25th anniversary one. I'll try it later

**Comment by NickMarkick** [comment]
![image](https://github.com/user-attachments/assets/24c9cb92-c902-4048-9fd1-f6dcac08c4c1)
Replacing the client map with the zip one worked, thanks!
